.gitignore

```diff
@@ -3,4 +3,5 @@ dist
 !src/**/*.js
 !src/**/*.flow
 !test/**/*.js
-!test/**/*.flow
\ No newline at end of file
+!test/**/*.flow
+node_modules/
```

Comment:
I don't mind adding this, however it probably belongs in your `~/.gitignore` instead.